lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -758,15 +758,9 @@ def add_mc_directory_trans_id(xml, credit_card, three_d_secure)
       end
 
       def add_mc_ucafind(xml, credit_card, three_d_secure, options)
-        return unless three_d_secure
-
-        if options[:alternate_ucaf_flow]
-          return unless %w(4 6 7).include?(three_d_secure[:eci])
+        return unless three_d_secure && %w(4 6 7).include?(three_d_secure[:eci])
 
-          xml.tag! :UCAFInd, options[:ucaf_collection_indicator] if options[:ucaf_collection_indicator]
-        else
-          xml.tag! :UCAFInd, options[:ucaf_collection_indicator] || '4'
-        end
+        xml.tag! :UCAFInd, options[:ucaf_collection_indicator] if options[:ucaf_collection_indicator]
```

Comment:
What is the purpose of removing the `4` default? I hate random defaults like this so I like removing it, but just curious on the reasoning?

Reply:
That we should only be sending the filed if options[:ucaf_collection_indicator] at least that's how the original ticket was written for this.